libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -430,6 +430,24 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
 #endif // GPS_MAX_RECEIVERS > 1
 #endif // HAL_ENABLE_DRONECAN_DRIVERS
 
+#if AP_GPS_GSOF_ENABLED
+    // @Param: _GSOF_BAUD1
+    // @DisplayName: Baud rate for the first GSOF GPS 
+    // @Description: What baud rate to configure the first GSOF GPS to
+    // @Values: 7:115k, 11:230k
+    // @User: Advanced
```

Comment:
Perhaps this needs information that the flight controller needs a reboot if this is changed since the baud command is only sent during the constructor of the GPS instance?

Reply:
Reboot required is probably needed,